### PR TITLE
[FSTORE-1451] Merge pyproject from hsfs and hsml

### PR DESCRIPTION
**Pull request by aversey**
The next PR: https://github.com/logicalclocks/hopsworks-api/pull/228

The previous PR: https://github.com/logicalclocks/hopsworks-api/pull/227

### CODE REVIEW

File: python/hsml/model_registry.py [hunk 1]
```diff
@@ -19,10 +19,10 @@
 import humps
 from hsml import util
 from hsml.core import model_api
-from hsml.python import signature as python_signature  # noqa: F401
 from hsml.sklearn import signature as sklearn_signature  # noqa: F401
 from hsml.tensorflow import signature as tensorflow_signature  # noqa: F401
 from hsml.torch import signature as torch_signature  # noqa: F401
+from python.hsml.python import signature as python_signature  # noqa: F401
```
Comment (kennethmhc) [inline python/hsml/model_registry.py:25]: why only change the parent module of this?

Comment (aversey) [reply python/hsml/model_registry.py:25]: This is a mistake, which is reversed in the next PR. https://github.com/logicalclocks/hopsworks-api/pull/228/commits/0e24b88a9025d3880c46618b133c20a956ed513e

The problem is that for some reason, if you start python from repo's root, hsml.python is available only as python.hsml.python. In all other places, hsml.python can be imported without any problems.

Comment (kennethmhc) [reply python/hsml/model_registry.py:25]: can you cherry pick the commit to this PR

File: .github/workflows/mkdocs-main.yml [hunk 2]
```diff
@@ -21,10 +21,10 @@ jobs:
 
       - name: install deps
         working-directory: ./python
-        run: cp ../README.md . && pip3 install -r ../requirements-docs.txt && pip3 install -e .[dev]
+        run: cp ../README.md . && pip3 install -e .[dev]
```
Comment (kennethmhc) [inline .github/workflows/mkdocs-main.yml:24]: I think the requirements-docs.txt was trying to fix a CI problem: https://github.com/logicalclocks/hopsworks-api/pull/209

Comment (aversey) [reply .github/workflows/mkdocs-main.yml:24]: Ah, I see, thanks!

**Review by kennethmhc — APPROVED** [review]

**Pull request merged**